tests/ValueParsers/TimeParserTest.php

```diff
@@ -21,7 +22,7 @@ class TimeParserTest extends ValueParserTestBase {
 	 * @deprecated since 0.3, just use getInstance.
 	 */
 	protected function getParserClass() {
-		return 'ValueParsers\TimeParser';
+		throw new \LogicException( 'Should not be called, use getInstance' );
```

Comment:
If you introduce a backwards compatibility break, then why not remove the function?
This depends on the not yet released 3.0.0 of https://github.com/DataValues/Common but the dependency is not upgraded here.